Финальная ( стили\моб\таблет\декстоп

temuresanov01 opened the pull request:
поменял разметку для бордера ( десктоп ) и разметку в лишках для серого текста (теперь при изменения размера экрана текст переходит в новую строку ) по фиксил стили для всех экранов

temuresanov01 committed:
Финальная ( стили\моб\таблет\декстоп

YaroslavKnysh commented on src/sass/components/_program.scss:
```diff
@@ -139,155 +141,258 @@
     background-size: cover;
   }
 }
-@media (min-device-pixel-ratio: 2) and (min-width: 768px),
```
При 768 и 1314px остается разметка мобайл

temuresanov01 committed:
ПОПРАВКИ ПО СЕКЦИИ ПРОГРАММ

изправил переломы для мобилы -------- повесил фотки до класса лист-фото к первой лишки семестр1 через псевдо класс афтер ----------- в мобильной версии от центровал контент через флексы ---------- в таблет  выстовил по центру контент (поправил) через паддинг лефт

temuresanov01 committed:
правильные размеры фото десктопа для секции программ

Glitch458 reviewed (approved):
ok
